util/yaml/yaml.go

```diff
@@ -28,9 +28,11 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/runtime/serializer/streaming"
-	"k8s.io/apimachinery/pkg/util/yaml"
+	utilyaml "k8s.io/apimachinery/pkg/util/yaml"
```

Comment:
`utilyaml` alias is already used elsewhere for another package, and this might create confusion.
what about
```suggestion
	apiutil "k8s.io/apimachinery/pkg/util/yaml"
```